src/Parser.php

```diff
@@ -202,8 +203,7 @@ protected function parseFeature()
         \assert(\array_key_exists('keyword', $token));
         \assert(\array_key_exists('indent', $token));
 
-        $title = trim($token['value'] ?? '');
-        $description = null;
+        ['title' => $title, 'description' => $description] = $this->parseTitleAndDescription($token);
```

Comment:
Trimming and coalescing to & from null now all happens in `parseTitleAndDescription` depending on the compatibility mode.